README.md

```diff
@@ -2,9 +2,139 @@
 
 [![Documentation Status](https://readthedocs.com/projects/canonical-microceph/badge/?version=latest)](https://canonical-microceph.readthedocs-hosted.com/en/latest/?badge=latest)
 
-MicroCeph is snap-deployed Ceph with built-in clustering.
+<p align="center">
+<a href="https://snapcraft.io/microceph">MicroCeph</a> is snap-deployed Ceph with built-in clustering.
+</p>
 
-MicroCeph handles scaling out the monitor cluster, as well as automating placement and management of the manager and metadata processes in a Ceph cluster when units are dynamically added via easy-to-use tooling. It also helps with adding disks (OSDs) to the Ceph cluster.
+## Table of Contents
+1. [💡 Philosophy](#💡-philosophy)
+2. [🎯 Features](#🎯-features)
+3. [📖 Documentation](#📖-documentation)
+4. [⚡️ Quickstart](#⚡️-quickstart)
+5. [👍 How Can I Contribute ?](#👍-how-can-i-contribute)
 
-Documentation: https://canonical-microceph.readthedocs-hosted.com/en/latest/
-Snap: https://snapcraft.io/microceph
+## 💡 Philosophy
+
+Deploying and operating a Ceph cluster is complex because Ceph is designed to be a general purpose storage solution. This is a significant overhead for smaller Ceph clusters. [MicroCeph](https://snapcraft.io/microceph) solves this by being _opinionated_ and _focused_ at small scale. With MicroCeph, deploying and operating a Ceph cluster is as easy as a [Snap!](https://snapcraft.io/microceph)
+
+## 🎯 Features
+
+1. Quick and Consistent deployment with minimal overhead.
+2. Single-command operations (for bootstrapping, adding OSDs, service enablement etc).
+3. Isolated from host and upgrade-friendly.
+4. Built-in clustering so you don't have to worry about it!
+5. Tailored for small scale (or just your Laptop).
+
+## 📖 Documentation
+
+Refer to the [QuickStart](#⚡️-quickstart) section for your first setup. If you want to read _official_ _documentation_, please visit our hosted [Docs](https://canonical-microceph.readthedocs-hosted.com/en/latest/).
+
+## ⚡️ Quickstart
+
+### ⚙️ Installation and Bootstrapping Ceph cluster
+```bash
+# Install MicroCeph
+$ sudo snap install microceph
+
+# Bootstrapping the Ceph Cluster
+$ sudo microceph cluster bootstrap
+$ sudo microceph.ceph status
+    cluster:
+        id:     c8d120af-d7dc-45db-a216-4340e88e5a0e
+        health: HEALTH_WARN
+                OSD count 0 < osd_pool_default_size 3
+    
+    services:
+        mon: 1 daemons, quorum host (age 1m)
+        mgr: host(active, since 1m)
+        osd: 0 osds: 0 up, 0 in
+    
+    data:
+        pools:   0 pools, 0 pgs
+        objects: 0 objects, 0 B
+        usage:   0 B used, 0 B / 0 B avail
+        pgs: 
+```
+![Dashboard](/assets/bootstrap.png)
+
+> **_NOTE:_**
+You might've noticed that Ceph cluster is not _functional_ yet, We need OSDs!<br>
+But before that, If you are only interested in deploying on a single node, It would be worthwhile to change the CRUSH rules.
```

Comment:
Good addition -- may I suggest to add some explanation around the commands, maybe something

```
... it would be worthwhile to change the CRUSH rules. With the below commands, we're re-creating the default rule to have a failure domain of osd (instead of the default host failure domain)
```